client/actions/eventsPlanning/ui.js

```diff
@@ -313,6 +314,37 @@ const storeFilter = (filterId) => ({
     payload: filterId,
 });
 
+const fetchEventTemplates = () => (dispatch, getState, {api}) => {
```

Comment:
These are api calls for events only, so these 4 actions would be better placed in client/actions/events/api.js